### PR TITLE
Provide alternate API for normal distribution functions, deprecate existing API

### PR DESCRIPTION
**Pull request by jmh530**
As discussed [here](https://github.com/libmir/mir-algorithm/commit/a828eb1da6832fd4d57794141663deb09d58085d#r70070443)

### CODE REVIEW

File: source/mir/ediff.d [hunk 1]
```diff
@@ -931,8 +939,21 @@ Constructs the cumulative normal distribution function of the expression
 Params:
     value = expression
 +/
+deprecated("normalDistribution renamed, use normalCDF instead")
 auto normalDistribution(T)(const T value)
```
Comment (9il) [inline source/mir/ediff.d:943]: Alias works better for renaming:
```
deprecated("normalDistribution renamed, use normalCDF instead")
alias normalDistribution = normalCDF;
```

File: source/mir/math/func/normal.d [hunk 2]
```diff
@@ -82,19 +100,76 @@ T normalDistribution(T)(const T a)
...
 /// ditto
+deprecated("normalDistribution renamed, use normalCDF instead")
 T normalDistribution(T)(const T x, const T mean, const T stdDev)
     if (isFloatingPoint!T)
 {
     return normalDistribution((x - mean) / stdDev);
 }
 
-version(mir_test)
+version(mir_test_deprecated)
```
Comment (9il) [inline source/mir/math/func/normal.d:110]: We don't need to test deprecated API. Test should test only new names. Aliases ensures code is valid.

Comment (jmh530) [reply source/mir/math/func/normal.d]: Done

File: source/mir/ediff.d (hunk 1 above)
Comment (jmh530) [reply source/mir/ediff.d]: Done

**Pull request merged**